manticore/platforms/linux.py

```diff
@@ -67,16 +67,18 @@ def __getstate__(self):
         state = {}
         state['name'] = self.name
         state['mode'] = self.mode
+        state['closed'] = self.closed
         try:
-            state['pos'] = self.tell()
-        except (ValueError, IOError):
-            # This is to handle closed files and special files like /dev/tty
+            state['pos'] = None if self.closed else self.tell()
+        except IOError:
+            # This is to handle special files like /dev/tty
             state['pos'] = None
         return state
 
     def __setstate__(self, state):
         name = state['name']
         mode = state['mode']
+        closed = state['closed']
         pos = state['pos']
         self.file = file(name, mode)
```

Comment:
the original idea was to now use `closed` here to be able to check whether to call `file()` or not. but i'm now realizing that if we don't call `file()`, and just assign `self.file = None` or something, that will break most of the properties in this class and cause them to AttributeError or something. 

a solution to this might be to always try to call `file()`, but then if the file is closed, immediately call `.close()`. this way, we don't unnecessarily have a file open, but the various informational attributes like `.name` will still exist.